pages/common/bat.md

```diff
@@ -0,0 +1,23 @@
+# bat
+
+> Print and concatenate files. A cat(1) clone with syntax highlighting and Git integration.
```

Comment:
Please move the second sentence to a subheading. Wrap cat within backticks and remove `(1)`.

Reply:
Sure :)